### PR TITLE
Update redis keyword to valkey in some sentinel functions (Redis Legacy)

### PR DESCRIPTION
**Pull request by hwware**
This PR updates all Redis/redis keywords to Valkey/valkey, including variable names, comments, function names. 

All sentinel test cases passed.

### CODE REVIEW

**Review by PingXie — APPROVED** [review]
Nice! LGTM

File: src/sentinel.c [hunk 1]
```diff
@@ -1246,13 +1245,13 @@ void sentinelDisconnectCallback(const redisAsyncContext *c, int status) {
...
  * with the same ID already exists. */
 
-sentinelRedisInstance *createSentinelRedisInstance(char *name,
-                                                   int flags,
-                                                   char *hostname,
-                                                   int port,
-                                                   int quorum,
-                                                   sentinelRedisInstance *primary) {
-    sentinelRedisInstance *ri;
+sentinelValkeyInstance *createSentinelRedisInstance(char *name,
```
Comment (zuiderkwast) [inline src/sentinel.c]: Function name is still createSentinelRedisInstance. Change this too?

File: src/sentinel.c [hunk 2]
```diff
@@ -1350,7 +1349,7 @@ sentinelRedisInstance *createSentinelRedisInstance(char *name,
  * This function does not take care of unlinking the instance from the main
  * primaries table (if it is a primary) or from its primary sentinels/replicas table
  * if it is a replica or sentinel. */
-void releaseSentinelRedisInstance(sentinelRedisInstance *ri) {
+void releaseSentinelRedisInstance(sentinelValkeyInstance *ri) {
```
Comment (zuiderkwast) [inline src/sentinel.c]: Function name still contains redis.

File: src/sentinel.c [hunk 3]
```diff
@@ -1448,10 +1447,10 @@ int removeMatchingSentinelFromPrimary(sentinelRedisInstance *primary, char *runi
  *
  * runid or addr can be NULL. In such a case the search is performed only
  * by the non-NULL field. */
-sentinelRedisInstance *getSentinelRedisInstanceByAddrAndRunID(dict *instances, char *addr, int port, char *runid) {
+sentinelValkeyInstance *getSentinelRedisInstanceByAddrAndRunID(dict *instances, char *addr, int port, char *runid) {
```
Comment (zuiderkwast) [inline src/sentinel.c]: Function name contains redis.

**Review by zuiderkwast — APPROVED** [review]
LGTM

I see some more occurrences of redis that can be changed, but that can be done in another PR if you want.

**Review by enjoy-binbin — COMMENTED** [review]
https://github.com/valkey-io/valkey/blob/5925cc7c4fdd89e9547fa2602d5e46213c70f891/src/sentinel.c

i see there are still some places are using the Redis word, can you do a search and also fix it? like addReplySentinelRedisInstance

File: src/sentinel.c [hunk 4]
```diff
@@ -241,10 +241,10 @@ typedef struct sentinelValkeyInstance {
...
     int failover_state;      /* See SENTINEL_FAILOVER_STATE_* defines. */
     mstime_t failover_state_change_time;
-    mstime_t failover_start_time;                   /* Last failover attempt start time. */
-    mstime_t failover_timeout;                      /* Max time to refresh failover state. */
-    mstime_t failover_delay_logged;                 /* For what failover_start_time value we
-                                                       logged the failover delay. */
+    mstime_t failover_start_time;                    /* Last failover attempt start time. */
+    mstime_t failover_timeout;                       /* Max time to refresh failover state. */
+    mstime_t failover_delay_logged;                  /* For what failover_start_time value we
+logged the failover delay. */
```
Comment (zuiderkwast) [inline src/sentinel.c]: Clang-format accepts this? The comment continues on the next line without indentation looks very wrong to me.

**Review by zuiderkwast — APPROVED** [review]
Thanks! Just one issue I wonder if clang-format doesn't understand.

File: src/sentinel.c (hunk 4 above)
Comment (hwware) [reply src/sentinel.c]: yes, Clang-format accepts it. The result is due to clang-format, not me.

Comment (zuiderkwast) [reply src/sentinel.c]: ... but it also accepts this?

```suggestion
    mstime_t failover_delay_logged;                  /* For what failover_start_time value we
                                                      * logged the failover delay. */
```

Comment (hwware) [reply src/sentinel.c]: it looks like clang-format accepts both. But for your convivence,  I update the format to much more readable.

**Pull request merged**